lib/chef-analytics/token_manager.rb

```diff
@@ -20,10 +20,7 @@ module TokenManager
     def fetch_token(options = {})
       identity = ChefAnalytics::Identity.new(options)
       token = identity.token
-      if token.nil?
-        ui.error 'Couldn\'t get OAuth2 token from OC-ID server'
-        exit 1
-      end
+      fail 'Couldn\'t get OAuth2 token from OC-ID server' if token.nil?
```

Comment:
I think if we continue adding to this repo we should definitely do that - but for now I am just trying to make as small of changes as possible.